Sources/Plasma/FeatureLib/pfPython/cyAccountManagementGlue.cpp

```diff
@@ -64,7 +64,7 @@ PYTHON_GLOBAL_METHOD_DEFINITION(PtCreatePlayer, args, "Params: playerName, avata
     char* playerName;
     char* avatarShape;
     char* invitation;
-    if (!PyArg_ParseTuple(args, "ssz", &playerName, &avatarShape, &invitation))
+    if (!PyArg_ParseTuple(args, "etetz", "utf8", &playerName, "utf8", &avatarShape, &invitation))
```

Comment:
The `"et"` format allocates memory, which must be freed with `PyMem_Free`.  At least in Python 2.x, the `"s"` format can also accept unicode strings, which will get automatically converted to 8-bit using PyUnicode's default encoding (`sys.setdefaultencoding`) and are cached within the unicode object (so there's no need to explicitly free it).

Or even better, why not use `"O"` with `PyString_CheckEx` and `PyString_AsStringEx` which were explicitly created to handle conversion of `str` and `unicode` to `ST::string`?